examples/layer-browser/src/examples/core-layers.js

```diff
@@ -168,6 +168,7 @@ const ScatterplotLayerExample = {
     getRadius: d => get(d, 'SPACES'),
     opacity: 0.5,
     pickable: true,
+    pickingHighlightColor: [1, 0, 0, 1],
```

Comment:
For highlight color, picking module doesn't scale the color from [0-255] to [0-1], we might need to fix that then we can specify colors in 0-255 range.